lib/templates/dialogue.genie

```diff
@@ -91,6 +91,7 @@ context : S.ContextInfo = {
     ctx_without_projection,
     ctx_with_notification,
     ctx_for_related_question,
+    ctx_notification_for_reminder,
```

Comment:
What sets this context tag? If you want to add a context tag, you need to change tagContextForAgent or getContextPhrases in lib/templates/state_manip.ts